spec/feature/account_home_spec.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.describe "AccountHome", type: :feature do
+  before do
+    stub_content_store_has_item("/")
+  end
+
+  context "/account/home" do
```

Comment:
Yes, we're using the default rather than setting it explicitly (no need to do that), and in a system test we can just test if we've been bounced to the new URL (in which case a redirect has happened)